gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
```

Comment:
It would be better to update the Gradle version to `8.11.1` to make it consistent with other connectors

Reply:
Updated the gradle wrapper version